src/app/components/MediaPlayer/AudioLoader/index.jsx

```diff
@@ -32,7 +32,12 @@ const spin = keyframes`
 const Spinner = styled.svg`
   width: 64px;
   height: 64px;
+
   stroke: currentColor;
+  @media screen and (-ms-high-contrast: active) {
```

Comment:
Just had a look at the newer standards compliant replacement for `ms-high-contrast`: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors 

Just sometimes emotion translates these styles back to browser specific ones but can't see a mention of `force-colors` in the emotion codebase.

Reply:
Not supported in IE it seems - this media query is supported by both IE and Edge